CONTRIBUTING.rst

```diff
@@ -0,0 +1,73 @@
+Contribution Guidelines
+=======================
+
+We use `Github projects`_ to organize our ticket workflow.
+If there's `a particular issue`_ you would like to work on and it isn't already assigned, feel free to assign it to yourself and start work!
```

Comment:
It looks like there's not actually a way for us to let people assign tickets to themselves :-(